tests/ui/traits/new-solver/two-projection-param-candidates-are-ambiguous.rs

```diff
@@ -0,0 +1,30 @@
+// compile-flags: -Ztrait-solver=next
+
+// When we're solving `<T as Foo>::Assoc = i32`, we actually first solve
+// `<T as Foo>::Assoc = _#1t`, then unify `_#1t` with `i32`. That goal
+// with the inference variable is ambiguous when there are >1 param-env
+// candidates.
+
+// We don't unify the RHS of a projection goal eagerly when solving, both
+// for caching reasons and partly, to make sure that we don't make the new
```

Comment:
```suggestion
// for caching reasons and partly to make sure that we don't make the new
```
it feels like there shouldn't be a comma there. Or I guess if it is correct, why? :sweat_smile:

Reply:
No, it was a mistake.